<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDialog</class>
 <widget class="QDialog" name="ClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый клиент</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="maleRadioButton">
         <property name="text">
          <string>Мужской</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="femaleRadioButton">
         <property name="text">
          <string>Женский</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="surnameLabel">
         <property name="text">
          <string>Фамилия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="surnameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="phone1Label">
         <property name="text">
          <string>Телефон 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phone1LineEdit">
         <property name="inputMask">
          <string>+7 999 999 99 99</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="phone2Label">
         <property name="text">
          <string>Телефон 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phone2LineEdit">
         <property name="inputMask">
          <string>+7 999 999 99 99</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="emailLabel">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="directionLabel">
         <property name="text">
          <string>Направление</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="directionsComboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Адрес объекта (область, шоссе, город, улица, дом, корпус, кв.)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="addressPlainTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="passportGroupBox">
       <property name="title">
        <string>Паспорт</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="passportLabel">
          <property name="text">
           <string>Серия, №, кем и когда выдан, КП, дата и место рожд.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="passportPlainTextEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="paddressLabel">
          <property name="text">
           <string>Адрес регистрации</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="paddressPlainTextEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okPushButton">
         <property name="text">
          <string>Сохранить</string>
         </property>
         <property name="icon">
          <iconset resource="kcrm.qrc">
           <normaloff>:/images/ok32.png</normaloff>:/images/ok32.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>maleRadioButton</tabstop>
  <tabstop>femaleRadioButton</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>phone1LineEdit</tabstop>
  <tabstop>phone2LineEdit</tabstop>
  <tabstop>okPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="kcrm.qrc"/>
 </resources>
 <connections/>
</ui>
